### PR TITLE
Add item Controls sidebar in User Guide

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contains add of the text and images of the Controls sidebar item and delete unused images.

I made a description of this panel in the form of a table containing an icon and a description of each tool.

### How has this been tested?

N/A
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/documentation/user_guide.md [hunk 1]
```diff
@@ -814,6 +814,40 @@ Change the color scheme of annotation:
...
+|![](static/documentation/images/image148.jpg)|``Cursor`` (``Esc``)- a basic annotation pedacting tool.              |
+|![](static/documentation/images/image149.jpg)|``Move the image``- a tool for moving around the image without<br/> the possibility of editing.|
+|![](static/documentation/images/image150.jpg)|``Rotate``- two buttons to rotate the current frame<br/> a clockwise (``Ctrl+R``) and anticlockwise (``Ctrl+Shift+R``).<br/> You can enable ``Rotate all images`` in the settings to rotate all the images in the job
+
+**Zoom block** - contains tools for image zoom.
+|Icon                                         |Description                                                           |
+|--                                           |--                                                                    |
+|![](static/documentation/images/image151.jpg)|``Fit image``- fits image into the workspace size.<br/> Shortcut - double click on an image|
+|![](static/documentation/images/image166.jpg)|``Select a region of interest``- zooms in on a selected region.<br/> You can use this tool to quickly zoom in on a specific part of the frame.|
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:832]: image166 is missing

File: cvat/apps/documentation/user_guide.md [hunk 2]
```diff
@@ -814,6 +814,40 @@ Change the color scheme of annotation:
...
+
+**Shapes block** - contains all the tools for creating shapes.
+|Icon                                         |Description   |Links to section  |
+|--                                           |--            |--                |
+|![](static/documentation/images/image167.jpg)|``Rectangle``|[Shape mode](#shape-mode-basics); [Track mode](#track-mode-basics);<br/> [Drawing by 4 points](#annotation-with-rectangle-by-4-points)|
+|![](static/documentation/images/image168.jpg)|``Polygon``  |[Annotation with polygons](#annotation-with-polygons)  |
+|![](static/documentation/images/image169.jpg)|``Polyline`` |[Annotation with polylines](#annotation-with-polylines)|
+|![](static/documentation/images/image170.jpg)|``Points``   |[Annotation with points](#annotation-with-points)      |
+|![](static/documentation/images/image176.jpg)|``Cuboid``   |[Annotation with cuboids](#annotation-with-cuboids)      |
+|![](static/documentation/images/image171.jpg)|``Tag``      |                                                       |
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:842]: `Annotation with tags` link?

File: cvat/apps/documentation/user_guide.md [hunk 3]
```diff
@@ -814,6 +814,40 @@ Change the color scheme of annotation:
...
+**Zoom block** - contains tools for image zoom.
+|Icon                                         |Description                                                           |
+|--                                           |--                                                                    |
+|![](static/documentation/images/image151.jpg)|``Fit image``- fits image into the workspace size.<br/> Shortcut - double click on an image|
+|![](static/documentation/images/image166.jpg)|``Select a region of interest``- zooms in on a selected region.<br/> You can use this tool to quickly zoom in on a specific part of the frame.|
+
+**Shapes block** - contains all the tools for creating shapes.
+|Icon                                         |Description   |Links to section  |
+|--                                           |--            |--                |
+|![](static/documentation/images/image167.jpg)|``Rectangle``|[Shape mode](#shape-mode-basics); [Track mode](#track-mode-basics);<br/> [Drawing by 4 points](#annotation-with-rectangle-by-4-points)|
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:837]: link is dead `#annotation-with-rectangle-by-4-points`

File: cvat/apps/documentation/user_guide.md [hunk 4]
```diff
@@ -814,6 +814,40 @@ Change the color scheme of annotation:
...
+|![](static/documentation/images/image170.jpg)|``Points``   |[Annotation with points](#annotation-with-points)      |
+|![](static/documentation/images/image176.jpg)|``Cuboid``   |[Annotation with cuboids](#annotation-with-cuboids)      |
+|![](static/documentation/images/image171.jpg)|``Tag``      |                                                       |
+
+**Edit block** - contains tools for editing tracks and shapes.
+|Icon                                         |Description                                        |Links to section  |
+|--                                           |--                                                 |--                |
+|![](static/documentation/images/image172.jpg)|``Merge Shapes``(``M``) — starts/stops the merging shapes mode.  |[Track mode (basics)](#track-mode-basics)|
+|![](static/documentation/images/image173.jpg)|``Group Shapes`` (``G``) — starts/stops the grouping shapes mode.|[Shape grouping](#shape-grouping)|
+|![](static/documentation/images/image174.jpg)|``Split`` — splits a track.                                      |[Track mode (advanced)](#track-mode-advanced)|
```
Comment (nmanovic) [inline cvat/apps/documentation/user_guide.md:849]: the link is dead `#track-mode-advanced`

**Comment by TOsmanov** [comment]
@nmanovic , some links will work after updating the relevant sections.

image070.jpg, image071.jpg, image103.jpg - context menu is not available in the new UI

image101.jpg - in the new UI, located in the settings section

image105.jpg, image108.jpg, image118.jpg - unused images

**Comment by nmanovic** [comment]
@TOsmanov , could you please resolve conflicts in the patch?

**Comment by nmanovic** [comment]
@TOsmanov , some links are dead. Could you please check?

**Pull request merged**